dev/Dockerfile.dev

```diff
@@ -10,7 +10,7 @@ COPY dev/requirements_development.txt dev/.
 
 RUN pip install --no-cache-dir --requirement dev/requirements_development.txt
 
-COPY dev/wait.sh dev/.
+COPY wait.sh ./
```

Comment:
Why did you move the file? I think it belongs in `/dev/`

Reply:
It is not used in the main Dockerfile